zenodo/modules/spam/views.py

```diff
@@ -105,6 +105,87 @@ def delete(user_id):
         ctx.update(records=records)
         return render_template('zenodo_spam/delete.html', **ctx)
 
+def user_info(owners):
+    users = (
+        User.query
+        .options(db.joinedload(User.profile), db.joinedload(
+            User.external_identifiers))
+        .filter(User.id.in_(owners))
+    )
+
+    data = {}
+
+    for user in users:
+        info = {
+            "id": user.id,
+            "email": user.email,
+            "full_name": user.profile.full_name,
+            "username": user.profile.username,
+            "last_records": "",
+            "external_identifiers": "",
+        }
+
+        # TODO maybe make it more beautiful?
+        try:
+            if user.external_identifiers:
+                for identifier in user.external_identifiers:
+                    info["external_identifiers"] = (
+                        info["external_identifiers"] + identifier.method + ", "
+                    )
+                info["external_identifiers"] = info["external_identifiers"][
+                    :-2
+                ]
+        except Exception:
+            pass
+
+        last_records = owners[user.id]['last_records']
+        if last_records:
+            for record in last_records:
+                info["last_records"] = (
+                    info["last_records"] + record.title + ", "
```

Comment:
There are some records with very long titles which might cause some display issues, so we might get some funny renderings.

Reply:
I thought about cutting the title short, eg, display the first 20 chars of each title, will update it